lib/ModelMixins/AsyncMappableMixin.ts

```diff
@@ -29,7 +30,8 @@ function AsyncMappableMixin<T extends Constructor<Model<MappableTraits>>>(
      * If the map items are already loaded or already loading, it will
      * return the existing promise.
      */
-    loadMapItems(): Promise<void> {
+    async loadMapItems(): Promise<void> {
+      await this.showInitialMessageIfRequired();
```

Comment:
I'm not sure this is the best place for this. What about putting something like:
```ts
if (ShowableMixin.isMixedInto(item) {
   await item.showInitialMessageIfRequired();
}
```
in https://github.com/TerriaJS/terriajs/blob/6c205d9fcb48857b098993e5723dedd5a5e24683/lib/Models/Workbench.ts#L195-L201

Reply:
We discussed this, you were right